action.yaml

```diff
@@ -0,0 +1,96 @@
+---
+name: "Presidio client - execute analyze action"
+description: "Github Action to execute Presidio framework using presidio_cli"
+inputs:
+  verify-path:
```

Comment:
Might as well call it `path`. Keep it simple since this is an exclusive action.

Reply:
changed